src/ert/gui/tools/plot/widgets/clearable_line_edit.py

```diff
@@ -70,29 +72,29 @@ def hidePlaceHolder(self) -> None:
             palette.setColor(self.foregroundRole(), self._active_color)
             self.setPalette(palette)
 
-    def focusInEvent(self, focus_event):
-        QLineEdit.focusInEvent(self, focus_event)
+    def focusInEvent(self, a0: Optional[QFocusEvent]) -> None:
+        QLineEdit.focusInEvent(self, a0)
         self.hidePlaceHolder()
 
-    def focusOutEvent(self, focus_event):
-        QLineEdit.focusOutEvent(self, focus_event)
+    def focusOutEvent(self, a0: Optional[QFocusEvent]) -> None:
+        QLineEdit.focusOutEvent(self, a0)
         if not QLineEdit.text(self):
             self.showPlaceholder()
 
-    def keyPressEvent(self, key_event):
-        if key_event.key() == Qt.Key_Escape:
+    def keyPressEvent(self, a0: Optional[QKeyEvent]) -> None:
+        if a0 is not None and a0.key() != Qt.Key.Key_Escape:
             self.clear()
             self.clearFocus()
-            key_event.accept()
+            a0.accept()
 
-        QLineEdit.keyPressEvent(self, key_event)
+        QLineEdit.keyPressEvent(self, a0)
 
-    def setText(self, string: Optional[str]) -> None:
+    def setText(self, a0: Optional[str]) -> None:
         self.hidePlaceHolder()
 
-        QLineEdit.setText(self, string)
+        QLineEdit.setText(self, a0)
 
-        if len(str(string)) == 0 and not self.hasFocus():
+        if len(str(a0)) == 0 and not self.hasFocus():
```

Comment:
Should this have been 'text' instead, since this is not an event?

Reply:
The setText function in QLineEdit names the first parameter `a0` and sets the type to `Optional[str]`, so that is what we have to use.